api/image.go

```diff
@@ -325,3 +351,161 @@ func (api *API) PublishImageHandler(w http.ResponseWriter, req *http.Request) {
 	}
 	api.publishedProducer.ImagePublished(&event)
```

Comment:
I see that this has not been updated as part of this PR, but there is an unhandled error here. (Maybe you would want to look at this separately from this PR)

Reply:
I think this should be handled as part of https://github.com/ONSdigital/dp-image-api/pull/25, which modifies publish